pom.xml

```diff
@@ -43,7 +48,9 @@
                 <plugin>
                     <groupId>org.jenkins-ci.tools</groupId>
                     <artifactId>maven-hpi-plugin</artifactId>
-                    <version>2.0</version>
+                    <configuration>
+                        <pluginFirstClassLoader>true</pluginFirstClassLoader>
```

Comment:
You forgot this, it seems.

Reply:
ha I removed it temporarily since the way I had it originally was causing errors so it wouldn't be possible for someone downloading this repo to even select the project for post build